src/app/client/src/app/modules/core/services/form/form.service.ts

```diff
@@ -57,7 +57,11 @@ export class FormService {
               subType: this.configService.appConfig.formApiTypes[formInputParams.contentType]
               ? this.configService.appConfig.formApiTypes[formInputParams.contentType]
               : formInputParams.contentType,
-              rootOrgId:this.configService.appConfig.frameworkCatConfig.changeChannel ? this.configService.appConfig.frameworkCatConfig.channel : hashTagId ? hashTagId : rootOrgId,
+              rootOrgId: this.configService.appConfig.frameworkCatConfig && this.configService.appConfig.frameworkCatConfig.changeChannel
```

Comment:
@sedin-tushar why you have added this condition this.configService.appConfig.frameworkCatConfig &&

Reply:
using `this.configService.appConfig.frameworkCatConfig && `-  testcases are passing , using `rootOrgId:this.configService.appConfig.frameworkCatConfig.changeChannel ? ` the test cases are failing due to undefined error 
 

> shall i revert this change and do the appropiate changes in test case and commit ?

Reply:
Please add the fix inside the test case not in the code

Reply:
Sure , will resolve this